tests/test_cookiejar.py

```diff
@@ -272,26 +272,32 @@ async def test_domain_filter_ip_cookie_receive(cookies_to_receive) -> None:
     assert len(jar) == 0
 
 
-async def test_preserving_ip_domain_cookies(loop) -> None:
-    jar = CookieJar(unsafe=True)
-    jar.update_cookies(SimpleCookie(
-        "shared-cookie=first; "
-        "ip-cookie=second; Domain=127.0.0.1;"
-    ))
-    cookies_sent = jar.filter_cookies(URL("http://127.0.0.1/")).output(
-        header='Cookie:')
-    assert cookies_sent == ('Cookie: ip-cookie=second\r\n'
-                            'Cookie: shared-cookie=first')
-
-
-async def test_preserving_quoted_cookies(loop) -> None:
-    jar = CookieJar(unsafe=True)
+@pytest.mark.parametrize(
+    'cookies,expected,quote_bool', [
+        ("shared-cookie=first; ip-cookie=second; Domain=127.0.0.1;",
+         'Cookie: ip-cookie=second\r\nCookie: shared-cookie=first', True),
+        ("ip-cookie=\"second\"; Domain=127.0.0.1;", 'Cookie: ip-cookie=\"second\"', True),
+        ("custom-cookie=value/one;", 'Cookie: custom-cookie="value/one"', True),
+        ("custom-cookie=value1;", 'Cookie: custom-cookie=value1', True),
+        ("custom-cookie=value/one;", 'Cookie: custom-cookie=value/one', False),
+    ]
```

Comment:
One way to set IDs is this:
```suggestion
    ],
    ids=(
        'IP domain preserved',
        'no shared cookie',
        'quoted cookie with special char',
        'quoted cookie w/o special char',
        'unquoted cookie with special char',
    ),
```
Alternatively, you could use `pytest.param(..., id=...)` with each of the params inline.

Reply:
Applied. Fixed a sneaky `space` that got inserted in a previous commit at the same time.